src/analysis/Environments/Environments.ts

```diff
@@ -762,12 +760,13 @@ export const Environments: React.FC<EnvironmentsProps> = (props) => {
                   const cloudEnvironment = filteredCloudEnvironments[rowIndex];
                   // We assume that all apps get created in zone 'us-central1-a'.
                   // If zone or region is not present then cloudEnvironment is an app so we return 'us-central1-a'.
-                  const location = isRuntime(cloudEnvironment)
-                    ? isGceConfig(cloudEnvironment.runtimeConfig) || isGceWithPdConfig(cloudEnvironment.runtimeConfig)
-                      ? cloudEnvironment.runtimeConfig.zone
-                      : _.toLower(getNormalizedComputeRegion(cloudEnvironment.runtimeConfig))
-                    : defaultComputeZone.toLowerCase();
-                  return location;
+                  if (isApp(cloudEnvironment)) {
```

Comment:
Could you update the comment above? We now return the region for both apps and runtimes, there are no more default 'us-central1-a' zones returned

Reply:
Removed the comment since it doesn't seem to be needed anymore